test/builtin/test_makeboxes.py

```diff
@@ -17,6 +17,89 @@ def skip_or_fail(x):
     skip_or_fail = pytest.mark.xfail
 
 
+@pytest.mark.parametrize(
+    ("str_expr", "str_expected", "fail_msg"),
+    [
+        ("MakeBoxes[x]", '"x"', "StandardForm"),
+        (
+            "MakeBoxes[InputForm[x]]",
+            'InterpretationBox[StyleBox["x", ShowStringCharacters -> True, NumberMarks -> True], InputForm[x], Editable -> True, AutoDelete -> True]',
```

Comment:
WMA has `x`, instead of `InputForm[x]` and there are no quotes around "x" in `StyleBox["x"]`.  What gives?